Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2748,6 +2748,9 @@ public func matmul<Scalar: Numeric>(
     _ rhs: Tensor<Scalar>,
     transposed transposeRhs: Bool = false
 ) -> Tensor<Scalar> {
+    precondition(lhs.rank >= 2, "The input must have rank greater than or equal to 2.")
+    precondition(rhs.rank >= 2, "The input must have rank greater than or equal to 2.")
+    precondition(lhs.shape[1] == rhs.shape[0] , "Dimensions must be equal,but are \(lhs.shape[1]) and \(rhs.shape[0]).")
```

Comment:
Nit: space after comma.

Reply:
Does space have any effect on output??

Reply:
Not exactly, that's just a formatting suggestion.